src/main/java/internetShop/category/FoodCategory.java

```diff
@@ -0,0 +1,12 @@
+package internetShop.category;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+@AllArgsConstructor
+public class FoodCategory implements Category, Serializable {
+    private String expirationDate;
```

Comment:
in all classes which implemented the serialization, you should implement the UUID filed otherwise you might have a serialization exception

Reply:
https://stackoverflow.com/questions/12912287/intellij-idea-generating-serialversionuid